### ISSUE TITLE
Doesn't seem to work

### ISSUE THREAD

**Issue by bbugh** [BetterErrors/better_errors-pry#4]
I wish I had a better message to report than "it doesn't work", but there's no feedback or error message.

* I added the gem to the Gemfile as specified (in development) in the correct order
* I double checked that I'm running in development, 
* updated the `binding_of_caller` and `better_errors` gems because they were a little out of date, 
* stopped spring, 
* stopped the server, 
* tried straight puma, 
* tried `rails s`, 
* reset the terminal, 
* reset the computer

... nothing. When better_errors loads, I still have the regular old boring non-Pry console. 

Is there some kind of other installation or initializer that needs to be set up that isn't mentioned in the Readme?

**Comment by RobinDaugherty** [comment]
Sorry @bbugh I was missing some documentation. This gem doesn't implicitly activate Pry in BetterErrors–BetterErrors does not currently support doing that. So you must still activate pry: https://github.com/charliesome/better_errors/wiki/Pry

I have a branch where I started to make this automatic, but it relies on changes I need to make in BetterErrors first, and I've been too busy to finish all of it. I've updated the readme in the meantime.
